app.go

```diff
@@ -335,6 +344,11 @@ func New(opts ...Option) *App {
 		return app
 	}
 
+	if app.visualize {
```

Comment:
This should be moved up since, since there is error handling above this statement that should include this process.

Reply:
Removed the boolean implementation.

Reply:
Removed.